source/server/getting-started/configuration/index.rst

```diff
@@ -2,81 +2,104 @@
+from file.
 
-It is possible to modify these configs through the in-game command ``/sponge config``. The syntax for the config
+Overriding the Global Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Say you would like to override any of the global configs in just a single world or dimension type, for example raise
+the spawn cap of monsters in just the Vanilla Nether. Subconfigs allow you to do just that, inside the main Sponge
+config folder you can find a ``worlds`` folder. First sorted by mods, then dimension type, you can find configs for
+every world and dimension on your server. Simply add whatever you want to change to those configs and it will just
+apply in the related worlds. Overriding any less specific configs.
+This follows a simple order: World > Dimension > Global
+
+Modifying the Configuration Using Commands
+==========================================
+
+It is possible to modify most configs through the in-game command ``/sponge config``. The syntax for the config
 command looks like this:
 
 .. code-block:: none
 
     /sponge config <flag> <key> <value>
 
-There are *flags* for specifying the target that you would wish to change. These flags are global, dimension, and world.
+There are the following **flags** for specifying the target that you would wish to change.
 
-* ``-g`` is the flag for global
-* ``-d <dim>`` targets a dimension (replacing *<dim>* with the dimension you want to configure)
-* ``-w <world>`` targets one world (replacing *<world>* with your chosen world).
+* ``-g`` is the flag for global (``global.conf``)
+* ``-d <dim>`` targets a dimension (``dimension.conf``)
+* ``-w <world>`` targets one world (``world.conf``)
+* ``-t`` is the flag for tracker (``tracker.conf``)
 
-The *key* is the value you want to change. The *value* is whatever you want to change the value of the key to.
+``<name>`` being a placeholder, you need replace with the name of your target, e.g. ``-d minecraft:nether``.
 
-Here is an example of this command in action:
+The **key** is the value you want to change. The **value** is whatever you want to change the value of the key to.
+Say you have the following file ``config/sponge/minecraft/nether/dimension.conf`` and want to change `chunk-load`
+to `true`.
 
 .. code-block:: none
 
-    /sponge config -d minecraft:nether logging.chunk-load true
+    sponge {
+        logging {
+            # Log when chunks are loaded
+            chunk-load=false
+        }
+    }
 
-This will set the config to log when chunks are loaded for the nether.
+Here is the command you would use:
 
-If you need to check the value of a key, you would need to omit the *value*. Checking the value of a key such as
+.. code-block:: none
+
+    /sponge config -d minecraft:nether logging.chunk-load true
+
+If you need to check the value of a key, you would need to omit the value. Checking the value of a key such as
```

Comment:
Replace "would need to" with "simply". 

_This bit is more a note to self: There is persistent use of weird tense in this document ("would need to") instead of simpler stuff (should, will, is, etc). It makes it longer and read oddly, so I should clean that up sometime._

Reply:
yeah I think this is a leftover from the original text